src/chunk.c

```diff
@@ -2266,6 +2266,18 @@ ts_chunk_get_by_relid(Oid relid, bool fail_if_not_found)
 
 	schema = get_namespace_name(get_rel_namespace(relid));
 	table = get_rel_name(relid);
+
+	if (schema == NULL || table == NULL)
```

Comment:
I am wondering if it is not better to have this check in `chunk_get_by_name` which is called by this function. That would cover other paths calling that function with NULL schema or NULL table name.

Reply:
`ts_chunk_get_by_relid` seems to be the only consumer of this `chunk_get_by_name` function right now. But yeah, on reflection, that seems a better location.

Reply:
Done. Shifted to `chunk_get_by_name`.